extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationOnSiteNotifications.text-formatter.js

```diff
@@ -15,10 +15,15 @@ define('ext.wikia.design-system.on-site-notifications.text-formatter', [
 
 			function fillArgs(message, args) {
 				return Object.keys(args).reduce(function (acc, key) {
-					return acc.replace('__' + key + '__', args[key])
+					return acc.replace('{' + key + '}', args[key])
 				}, message);
 			}
 
+			function getMessage(key) {
+				// TODO IRIS-4975 use mw.message(key).parse()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)